### PR TITLE
Feature/#21 Add theme support

### PR DESCRIPTION
**Pull request by MangriMen**

### CODE REVIEW

**Comment by MangriMen** [comment]
- [x] Fix warning with settings routes

**Comment by MangriMen** [comment]
[Add theme support](https://trello.com/c/MJbSkmkr/21-add-theme-support)

**Comment by MangriMen** [comment]
image

image

File: src/components/user/SettingSection.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export const SettingSection: FC<
+  BoxProps & { title: string; containerProps?: BoxProps }
+> = ({ title, containerProps, children }) => {
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+      <Typography variant="h3" color="primary.dark">
+        {title}
+      </Typography>
+      <ParDivider sx={{ width: '100%', marginBottom: '1rem' }} />
+      <Box {...containerProps}>{children}</Box>
```
Comment (StepanovRoma) [inline src/components/user/SettingSection.tsx:14]: sx={{ overflow: 'hidden' }}, I think it's much better this way, or write for every single in the props

**Review by StepanovRoma — CHANGES_REQUESTED** [review]

File: src/components/user/SettingSection.tsx (hunk 1 above)
Comment (StepanovRoma) [reply src/components/user/SettingSection.tsx:14]: image
screen with scroll bar

**Review by StepanovRoma — APPROVED** [review]

**Review by FFreestyler — APPROVED** [review]

**Pull request merged**